website/content/docs/post-processors/manifest.mdx

```diff
@@ -201,7 +201,7 @@ build.
 
 #!/bin/bash
 
-AMI_ID=$(jq -r '.builds[-1].artifact_id' manifest.json | cut -d ":" -f2)
+AMI_ID=$(jq -r '.builds[-1].artifact_id | split(": ") | .[1]' manifest.json)
```

Comment:
Good catch @lbajolet-hashicorp the extra space seems like a typo. I've tested both ways to ensure we did not mis anything against the example above. 
```suggestion
AMI_ID=$(jq -r '.builds[-1].artifact_id | split(":") | .[1]' manifest.json)
```